api/types/saml_test.go

```diff
@@ -39,3 +39,26 @@ func TestSAMLSecretsStrip(t *testing.T) {
 	require.Equal(t, withoutSecrets.GetSigningKeyPair().PrivateKey, "")
 	require.Equal(t, withoutSecrets.GetEncryptionKeyPair().PrivateKey, "")
 }
+
+// TestSAMLAcsUriHasConnector tests that the ACS URI has the connector ID as the last part if IdP-initiated login is enabled.
+func TestSAMLAcsUriHasConnectorName(t *testing.T) {
```

Comment:
```suggestion
func TestSAMLACSURIHasConnectorName(t *testing.T) {
```

I know it looks terrible, but idiom is to capitalize initialisms.